CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## v2.1.3
```

Comment:
I think this PR represents the full extent of the changes we want. So, we probably also want to bump the `version.txt` file to be able to release this code to the wild!

Reply:
You can tell I've been thinking of another project recently, misnaming `VERSION` as `version.txt`. I'm such an amateur!